pkg/state/state_ofed.go

```diff
@@ -36,8 +38,19 @@ import (
 	"github.com/Mellanox/network-operator/pkg/utils"
 )
 
-const stateOFEDName = "state-OFED"
-const stateOFEDDescription = "OFED driver deployed in the cluster"
+const (
+	stateOFEDName        = "state-OFED"
+	stateOFEDDescription = "OFED driver deployed in the cluster"
+
+	// TODO: update with new version
+	newMofedImageFormatVersion = "5.7-0.0.8.1"
```

Comment:
nit: could you please add a short comment which describes what this version means.
something like this: starting from this OFED version we should use new OFED image name format

Reply:
Updated,

i left the TODO as we still need to finalize the new version